pkg/sqlstore/impl/system_store_instrumented.go

```diff
@@ -393,3 +393,21 @@ func (s *InstrumentedSystemStore) InsertBlockExtraInfo(ctx context.Context, bloc
 
 	return err
 }
+
+// GetID returns node identifier.
+func (s *InstrumentedSystemStore) GetID(ctx context.Context) (string, error) {
+	start := time.Now()
+	id, err := s.store.GetID(ctx)
+	latency := time.Since(start).Milliseconds()
+
+	attributes := []attribute.KeyValue{
+		{Key: "method", Value: attribute.StringValue("Id")},
+		{Key: "success", Value: attribute.BoolValue(err == nil)},
+		{Key: "chainID", Value: attribute.Int64Value(int64(s.chainID))},
```

Comment:
This could be avoided since might be noisy for filtering logs per chain considering it's irrelevant.
Some side effects of not having the split store. (nre)